proposals/global-attributes-for-main.md

```diff
@@ -33,21 +33,23 @@ For example:
 Can be placed in any syntax tree of the compilation, and the `[STAThread]`
 attribute will be attached the entrypoint of the program.
 
-Both top level statements and regular entrypoints are supported by the attribute
-target. In the case of multiple entrypoints the attribute will only apply to the
-first chosen location.
+Only simple program entrypoints are supported. In the case of multiple
+entrypoints the attribute will only apply to the first chosen location.
```

Comment:
There are not multiple entry points for programs with top level statements.

Reply:
> Although its an error case, the compiler explicitly allows you to have multiple files with top level statements

It's an error case. The compiler explicitly does not allow it. Therefore, our handling of such an error case is an implementation detail, and doesn't belong in the specification.

Reply:
I think we could simplify this to essentially 

> Only simple program entrypoints are supported. It is not supported when there is an explicit `Main` method which qualifies as the entry point. 

I agree we should avoid listing out how it works in cases like multiple files with top level statements. That is already an error scenario, basically an invalid program. Generally we don't spec out behaviors for scenarios like that.